students/Sally_Shen/lesson02/assignment/src/charges_calc.py

```diff
@@ -0,0 +1,75 @@
+'''
+Returns total price paid for individual rentals 
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+
+
+def parse_cmd_arguments():
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument('-o', '--output', help='ouput JSON file', required=True)
+
```

Comment:
Don;t forget to add the -d debug argument to determine what level of logging to use!